app/controllers/admin/routes.py

```diff
@@ -62,7 +62,7 @@ def templateSelect():
 @admin_bp.route('/eventTemplates/<templateid>/create', methods=['GET','POST'])
 @admin_bp.route('/eventTemplates/<templateid>/<programid>/create', methods=['GET','POST'])
 def createEvent(templateid, programid=None):
-    if not g.current_user.isAdmin:
+    if not (g.current_user.isAdmin):
```

Comment:
The program manager should be able to create events for their programs, right? I thought this was already in there (and I see you are doing it down in edit event).